### ISSUE TITLE
Dont say confusing things when there are just unassigned interfaces

### ISSUE THREAD

**Issue by sgpinkus** [firewalld/firewalld#1182]
### What would you like to be added

I installed docker on a system with firewalld but no network manager. Now `firewall-cmd --list-services` and other commands warn me that:

> You're performing an operation over default zone ('public'),
> but your connections/interfaces are in zone 'docker' (see --get-active-zones)
> You most likely need to use --zone=docker option.

This message is really confusing. The implication is all my connections are in the docker zone but they're not. I just have interfaces that aren't assigned and are thus in the default zone. I didn't want --zone=docker.

It's not just a docker thing. It's any system that has at least one assigned interface and others that aren't, which I think would be common when not using network-manager.

Also, related to this, the output generated from `firewall-cmd --list-interfaces --zone=public` and `firewall-cmd --list-all-zones`  could make it much clearer that is what's happening too. It leaves doubt as to whether unassigned things are just by-passing the firewall altogether. Instead of listing no interfaces firewall-cmd could: 1. Enumerate unassigned interfaces and list them as being assigned to the default zone or just say "interfaces: ALL-UNASSIGNED-ONES" for the default zone.

### Why is this needed

Current output is confusing. One has to dig through the documentation to figure out how unassigned interfaces are handled. Even then the confusing output leave lingering doubt about it.

**Comment by erebion** [comment]
This also massively confuses me and that tends to happen every few months when I change something on my system. :/

Makes me think somehow `public` is no longer the zone I use and I screwed something up.

**Comment by sgpinkus** [comment]
> It's a firewalld principle that all unassigned traffic goes to the default zone.

Yeah I know :). I was suggesting the output of certain cli commands make it clear that is what's happening.
